src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../react-app-env';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<{ movies: Movie[]}> = ({
```

Comment:
In this case also will be better to create separate interface for props and use it. Because now you repeat types for your param and function, it's redundant.